### ISSUE TITLE
[Feature request]  The possibility of using only the host of that file in the prefix, instead of using all the permutations

### ISSUE THREAD

**Issue by TvMpt** [xnl-h4ck3r/GAP-Burp-Extension#36]
Hi,

When I select all the hosts from the sitemap and want to extract links (with the prefix with selected targets option) it seems to do all the combinations instead of just using the host from the file it's parsing. 

Thank you

**Comment by xnl-h4ck3r** [comment]
Hi @TvMpt. thanks for raising this
As discussed on Twitter, I think what would be best is to have the option of prefixing all selected target hosts selected in the sitemap (as it does currenltly), but the other option of only prefixing with the hosts of the response that the discovered path/filename was found in. 
So if we select `www.example.com`, `sub.example.com` and `api.example.com` in the sitemap, and the path `/test/config.txt` was only found in  `www.example.com` and  `sub.example.com` , then the original option (as it works now) would output:
```
https://www.example.com/test/config.txt
https://sub.example.com/test/config.txt
https://api.example.com/test/config.txt
```
But the new alternative option I need to add to GAP would just output:
```
https://www.example.com/test/config.txt
https://sub.example.com/test/config.txt
```
Does that sound right?
Thanks

**Comment by TvMpt** [comment]
Yes, a simple checkbox to switch between these functionality is more than enough. The problem here is not so much when there are subdomains, but when the main hosts are really different. This is for situations where you're using multiple targets in the same project.
